findimports.py

```diff
@@ -489,6 +572,12 @@ def parseFile(self, filename):
         module.imports = {
             self.findModuleOfName(imp.name, imp.level, filename, dir)
             for imp in module.imported_names}
+        if ignore_stdlib_modules:
+            module.imported_names = list(filter(
+                lambda info: info.name not in STDLIB_MODNAMES_SET,
+                module.imported_names,
+            ))
```

Comment:
I really dislike `filter(lambda ...)`.  If you don't want replace it with a list comprehension I'll do that myself after the merge.

Reply:
No worries, I didn't know exactly what you meant at the time. My bad